src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,17 @@
 import React from 'react';
 import './MoviesList.scss';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
-);
+import { MovieType } from '../../types/Movie';
+import { MovieCard } from '../MovieCard';
+
+type Props = {
+  movies: MovieType[];
+};
+
+export const MoviesList: React.FC<Props> = ({ movies }) => {
+  return (
+    <>
+      {movies.map(movie => (<MovieCard key={movie.imdbId} movie={movie} />))}
+    </>
```

Comment:
```suggestion
    <div className="movies">
      {movies.map(movie => (
        <MovieCard
           key={movie.imdbId}
           movie={movie} 
        />))}
    <div/>
```
You should take the whole BEM block that is for the movies, including the wrapping div.

Reply:
@CharAnagn  👍